plugins/ocm-backend/src/helpers/parser.ts

```diff
@@ -58,3 +60,41 @@ export const parseManagedCluster = (cluster: any): ClusterDetails => {
     ...parsedClusterInfo,
   };
 };
+
+export const parseUpdateInfo = (clusterInfo: any) => {
+  const versionAvailableUpdates =
+    clusterInfo.status.distributionInfo.ocp.versionAvailableUpdates;
+  const versionsAvailable =
```

Comment:
Why this name differs from the property name if the previous constant is named the same?

Can't you just unpack both from the object?

```ts
const { versionAvailableUpdates, versionsAvailable } = clusterInfo.status.distributionInfo.ocp;
```

Reply:
Resolved in the lastest commit.